cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/DcLineSegmentConversion.java

```diff
@@ -30,30 +34,78 @@ public DcLineSegmentConversion(PropertyBag l, Context context) {
         cconverter2 = context.dc().cgmesConverterFor(iconverter2);
     }
 
+    private void updateConverterStations(double pAC) {
+
+        // poleLossP is the active power loss at a DC Pole
+        // for lossless operation: P(DC) = P(AC) => lossFactor = 0
+        // for rectifier operation with losses: P(DC) = P(AC) - poleLossP => P(DC) / P(AC) = 1 - poleLossP1 / P(AC) = 1 - lossFactor
+        // for inverter operation with losses: P(DC) = P(AC) + poleLossP => P(AC) / P(DC) = 1 - poleLossP / P(DC) = 1 - poleLossP / (P(AC) + poleLossP) = 1 - lossFactor
+        double poleLossP1 = cconverter1.asDouble("poleLossP");
+        double poleLossP2 = cconverter2.asDouble("poleLossP");
+
+        float lossFactor = 0;
```

Comment:
I think that pAC is not the right value. I think, in the case of a good definition of pAC1 and pAC2, we should have something like that:

        double pAC1 = Double.isNaN(cconverter1.asDouble(TARGET_PPCC)) ? 0 : cconverter1.asDouble(TARGET_PPCC);
        double pAC2 = Double.isNaN(cconverter2.asDouble(TARGET_PPCC)) ? 0 : cconverter2.asDouble(TARGET_PPCC);

        float lossFactor1 = 0;
        if (decodeMode().equals(HvdcLine.ConvertersMode.SIDE_1_RECTIFIER_SIDE_2_INVERTER) && pAC1 != 0) {
            lossFactor1 = (float) (poleLossP1 / pAC1) * 100;
        } else if (decodeMode().equals(HvdcLine.ConvertersMode.SIDE_1_INVERTER_SIDE_2_RECTIFIER) && pAC1 + poleLossP1 != 0) {
            lossFactor1 = (float) (poleLossP1 / (pAC1 + poleLossP1)) * 100;
        }
        iconverter1.setLossFactor(lossFactor1);

        float lossFactor2 = 0;
        if (decodeMode().equals(HvdcLine.ConvertersMode.SIDE_1_INVERTER_SIDE_2_RECTIFIER) && pAC2 != 0) {
            lossFactor2 = (float) (poleLossP2 / Math.abs(pAC2)) * 100;
        } else if (decodeMode().equals(HvdcLine.ConvertersMode.SIDE_1_RECTIFIER_SIDE_2_INVERTER) && Math.abs(pAC2) + poleLossP2 != 0) {
            lossFactor2 = (float) (poleLossP2 / (Math.abs(pAC2) + poleLossP2)) * 100;
        }
        iconverter2.setLossFactor(lossFactor2);

But I am not sure when pAC1 or pAC2 is equal to NaN.